modules/spo-binding-workloads.adoc

```diff
@@ -51,7 +51,7 @@ $ oc label ns my-namespace spo.x-k8s.io/enable-binding=true
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc delete pods test-pod && oc create -f pod01.yaml
+$ oc delete pods test-pod && oc create -f test-pod
```

Comment:
instead of `Delete and re-create the pod to use the ProfileBinding object:`

could you change it to something like, create test-pod, and you need to recreate the pod if it is already exist for binding to work.
```
apiVersion: v1
kind: Pod
metadata:
  name: test-pod
spec:
  containers:
  - name: test-container
    image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21
```

Reply:
Good thinking @Vincent056 . I updated the PR. 
![image](https://github.com/openshift/openshift-docs/assets/46728411/7a671e9a-2473-43d7-86a8-e4da2aae719d)